airbyte-integrations/connectors/source-mysql/src/main/kotlin/io/airbyte/integrations/source/mysql/MysqlSourceOperations.kt

```diff
@@ -261,7 +261,7 @@ class MysqlSourceOperations :
             is And -> conj.flatMap { it.bindings() }
             is Or -> disj.flatMap { it.bindings() }
             is WhereClauseLeafNode -> {
-                val type = column.type as LosslessJdbcFieldType<*, *>
+                val type = /*column.type as LosslessJdbcFieldType<*, *>*/StringFieldType
```

Comment:
@xiaohansong this solves the immediate problem.
Since pk val is saved in mysql state as quoted string always we need to assume it is a string when parsed back to a query.

I'm sending this to discuss since:
a. I'm not sure who else is using the `WhereClauseNode.bindings()` extension.
b. Is string always going to work on any type of pk